demo/app/location-tracking/add-location-tracking.ts

```diff
@@ -0,0 +1,59 @@
+import Space from '../../../src/Space';
+import { colors } from '../utils/colors';
+
+export const addLocationTracking = (id: string, htmlElement: HTMLElement, space: Space) => {
+  const selectedTracker = space.locations.createTracker((locationChange) => locationChange.currentLocation === id);
+  const unselectedTracker = space.locations.createTracker(
+    (locationChange) => locationChange.previousLocation === id && locationChange.currentLocation !== id,
+  );
+
+  const selectedClasses = ['outline-2', 'outline-dashed'];
+
+  selectedTracker.on((change) => {
+    const self = space.getSelf();
+
+    htmlElement.setAttribute(`data-client-${change.member.clientId}`, 'true');
+
+    if (change.member.clientId === self.clientId) {
+      htmlElement.classList.add(...selectedClasses, 'outline-blue-400');
+      return;
+    }
+
+    const members = space.getMembers();
+
+    const memberIndex = members
+      .filter((member) => member.clientId === change.member.clientId)
+      .findIndex((member) => member.clientId === change.member.clientId);
+    const color = colors[memberIndex % colors.length];
+    const outlineColor = `outline-${color[0]}-${color[1]}`;
+    htmlElement.classList.add(...selectedClasses, outlineColor);
+  });
+
+  unselectedTracker.on((change) => {
```

Comment:
I think it would be nice to have the handlers as separate function so we easily show the usage of the location tracker API here, wdyt?